models/keep3r_network/ethereum/keep3r_network_ethereum_view_job_liquidity_log.sql

```diff
@@ -0,0 +1,166 @@
+{{ config (
+    materialized = 'view',
+    alias = 'view_job_liquidity_log',
+    post_hook = '{{ expose_spells(\'["ethereum"]\', "project", "keep3r", \'["wei3erHase", "agaperste"]\') }}'
+) }}
+
+WITH job_liquidities AS (
+
+    SELECT
+        ad.evt_block_time AS TIMESTAMP,
+        ad.evt_tx_hash AS tx_hash,
+        ad.evt_index,
+        'LiquidityAddition' AS event,
+        ad.contract_address AS keep3r,
+        ad._job AS job,
+        ad._liquidity AS token,
+        ad._amount / 1e18 AS amount
+    FROM
+        (
+            SELECT
+                evt_block_time,
+                evt_tx_hash,
+                evt_index,
+                contract_address,
+                _job,
+                _liquidity,
+                _amount
+            FROM
+                {{ source(
+                    'keep3r_network_ethereum',
+                    'Keep3r_evt_LiquidityAddition'
+                ) }}
+            UNION
+            SELECT
+                evt_block_time,
+                evt_tx_hash,
+                evt_index,
+                contract_address,
+                _job,
+                _liquidity,
+                _amount
+            FROM
+                {{ source(
+                    'keep3r_network_ethereum',
+                    'Keep3r_v2_evt_LiquidityAddition'
+                ) }}
+        ) ad
+    UNION ALL
+    SELECT
+        rm.evt_block_time AS TIMESTAMP,
+        rm.evt_tx_hash AS tx_hash,
+        rm.evt_index,
+        'LiquidityWithdrawal' AS event,
+        rm.contract_address keep3r,
+        rm._job job,
+        rm._liquidity AS token,- rm._amount / 1e18 AS amount
+    FROM
+        (
+            SELECT
+                evt_block_time,
+                evt_tx_hash,
+                evt_index,
+                contract_address,
+                _job,
+                _liquidity,
+                _amount
+            FROM
+                {{ source(
+                    'keep3r_network_ethereum',
+                    'Keep3r_evt_LiquidityWithdrawal'
+                ) }}
+            UNION
+            SELECT
+                evt_block_time,
+                evt_tx_hash,
+                evt_index,
+                contract_address,
+                _job,
+                _liquidity,
+                _amount
+            FROM
+                {{ source(
+                    'keep3r_network_ethereum',
+                    'Keep3r_v2_evt_LiquidityWithdrawal'
+                ) }}
+        ) rm
+),
+df AS (
+    SELECT
+        TIMESTAMP,
+        tx_hash,
+        evt_index,
+        event,
+        keep3r,
+        job,
+        token,
+        amount
+    FROM
+        job_liquidities
+    UNION
+    SELECT
+        migs.*,
```

Comment:
I think you should specify column names here since this goes into a UNION. If a new column is added to that model and a select * is used here then this view will fail.